src/SMU-Keysight_E3632A/main.py

```diff
@@ -90,7 +91,12 @@ def get_GUIparameter(self, parameter={}):
         # self.channel = int(parameter["Channel"])
 
         self.voltage_range = parameter["RangeVoltage"]
-
+        self.average = int(parameter["Average"])
+        
+        if self.average == 0:
+            msg = ("Average of 0 not possible. Disable average by setting it to 1.")
```

Comment:
The SMU GUI input for "Average" only allows values >1. It still makes sense to check for incorrect values of self.average (e.g. when the driver is used with pysweepme in a Python script instead of the SweepMe! GUI), but it could be generalized to check for self.average < 1

Reply:
Agreed, a change to self.average < 1 is sensible.